ext/standard/dns.c

```diff
@@ -47,6 +47,10 @@
 #endif
 #if HAVE_RESOLV_H
 #include <resolv.h>
+#if defined(__HAIKU__)
+extern void __res_ndestroy(res_state statp);
+#define res_ndestroy __res_ndestroy
+#endif
```

Comment:
This looks like a bit of a hack -- maybe we just need to include a different header? I found this one: https://github.com/haiku/haiku/blob/8f16317a5b6db5c672f331814273e5857555020f/src/system/libnetwork/netresolv/include/resolv.h

Reply:
that is not the same system wide the one you pinpoint is for building the network lib, the system resolv.h lacks those but maybe it will be resolved for the next beta 4 (more than one year between beta 2 and 3 so there is time :-))

Reply:
I see, let's go with this for now then...